### ISSUE TITLE
what is the image size for those pre-trained model?

### ISSUE THREAD

**Issue by SeanGuoXuan** [davidsandberg/facenet#223]
On the home page, there are a couple of pre-trained models '20170214-092102' and '20170216-091149'.  what is the input image size for those two? According the wiki 'Classifier training of inception resnet v1', I suppose it is 160 * 160. However, Inception_Resnnet-V1 requires an input image of size 299*299. In your file inception-resnet_v1.py, the first conv lay has output 149*149*32. How do you manage your architecture work with low resolution input than the preferred 299*299 input?

**Comment by DetionDX** [comment]
I have the same issue.

**Comment by DetionDX** [comment]
I have the same confusion. However, I think the 14914932 is the size of input image after first convolutional layer. but how to use the cropped 160 160 input image into 299 299 network?

**Comment by davidsandberg** [comment]
Since it's a CNN the input size is not that crucial and it can work fine with other image sizes as well. But of course, if the input images are too small the number of activations in the higher layers of the network will become to few and then it will crash. Also, if the input image size is changed the number of parameters in the resulting CNN will change as well, which means that training could require a different weight decay etc.
